Makefile

```
@@ -2,12 +2,13 @@ REPO=amazon-mws-analytics
 
 .PHONY: build
 build:
-	docker build -t $(REPO):latest .
+	docker build --target prod -t $(REPO):latest .
 
 .PHONY: style
 style:
 	docker run -it --rm -v "$(PWD)":/code jbbarth/black *.py
 
 .PHONY: lint
 lint:
-	docker run --rm -it -v "$(PWD)":/data cytopia/pylint *.py
+	docker build --target lint -t $(REPO):lint .
```

To go with the other suggestions:
```suggestion
	docker build --target tools -t $(REPO):lint .
```